rust-toolchain.toml

```diff
@@ -1,2 +0,0 @@
-[toolchain]
-channel = "1.74.0"
```

Comment:
accidentally deleted? at least for now the truth is, that meta-omnect has rust 1.74.0 and not e.g. latest 1.75.0

Reply:
ack, its in again